screens/Landmark.js

```diff
@@ -36,9 +36,19 @@ export default class LandmarkScreen extends React.Component {
   }
 
   render() {
-    const { landmarkDetails } = this.props;
+    const {
+      landmarkDetails: {
+        formatted_address,
+        formatted_phone_number,
+        location,
+        name,
+        opening_hours
+      }
+    } = this.props;
+
+    const address = formatted_address || location;
 
-    let description = landmarkDetails.description;
+    let description = this.props.landmarkDetails.description;
```

Comment:
why can't we pull description off of `this.props.landmarkDetails` like we do in the const above?

Reply:
I was trying not to change your code too much. Since we need to reassign description so it couldn't start out as a `const` but I can clean that up